pkg/certificate/certificate.go

```diff
@@ -90,9 +91,12 @@ func GenerateSignedCertificate(
 		return nil, nil, errors.New("while generating certificate: server name is empty")
 	}
 
+	SHA256serverNameEncrypted := sha256.New().Sum([]byte(serverName))
```

Comment:
The name of this variable only describes its technical content. It doesn't provide any functional or business information.

For example, if you name it websiteNameHash, it will make the reader understand that it is not the sha256 hash function that is important, but the fact of hashing the name of the website, to make it unique. This will also make this variable resistant to a potential change of the underlying hash function.

That being said, my suggestion is this:
```suggestion
	uniqueSiteNameID := sha256.New().Sum([]byte(serverName))
```